### ISSUE TITLE
Team update - Jun 29, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#141]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

**Updates from last week :heavy_check_mark:**
- #52. I heard I needed to make a PR for this, but it's already in jupyter/accessibility. Next step is where it lives in JupyterLab.
- #133. Got some community feedback on what might be in-demand for themes, thought nothing decisive came out of it. I need to update this issue with the notes.
- Notebook navigation user testing script is still in progress; I made some changes based on feedback this week. Also getting geared up to start recruiting testers.

**Challenges I faced and things I'd like assistance with 🙏**
- I don't know at the moment.

**My availability for next week**
- I'll be here all week.

**Important items for discussion 💬**
- Do we have a timeline for this sprint items and I'm missing it somewhere?

**Comment by gabalafou** [comment]
I'm sorry yall, I am totally swamped by the website work. We're trying to meet a July 6 internal deadline so that the site is launched by SciPy convention July 11.

**Thanks I'd like to give 🙌**
- Thanks to @trallard for helping take some of the pressure off.

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: nothing :(

**Challenges I faced and things I'd like assistance with 🙏**
- I mentioned this in yesterday's Jupyter accessibility call, but the CI/CD accessibility tooling is not as plug-and-play (for Gatsby/Next.js-style websites, at least) as I was hoping it might be.

**My availability for next week**
- Here all week! 

**Important items for discussion 💬**
- I've got nothing.

**Comment by trallard** [comment]
@isabela-pf 
	- will start curating WCAG/scope for the upcoming accessible themes 
	- @tonyfast add a PR with the scaffolding to - https://github.com/Quansight-Labs/jupyterlab-accessible-themes

	
💡 - Inspiration:
- https://github.com/gt-coar/jupyterlab-gt-coar-theme 
- https://github.com/dunovank/jupyter-themes
